homeassistant/components/apple_tv/strings.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "Apple TV",
+  "config": {
+    "flow_title": "Apple TV: {name}",
+    "step": {
+      "user": {
+        "title": "Setup a new Apple TV",
+        "description": "Start by entering the device name (e.g. Kitchen or Bedroom) or IP address of the Apple TV you want to add. If any devices were automatically found on your network, they are shown below.\n\nIf you cannot see your device or experience any issues, try specifying the device IP address.\n\n{devices}",
+        "data": {
+          "device_input": "Device"
+        }
+      },
+      "reconfigure": {
+        "title": "Device reconfiguration",
+        "description": "This Apple TV is experiencing some connection difficulties and must be reconfigured."
+      },
+      "pair_with_pin": {
+        "title": "Pairing",
+        "description": "Pairing is required for the `{protocol}` protocol. Please enter the PIN code displayed on screen. Leading zeros shall be omitted, i.e. enter 123 if the displayed code is 0123.",
+        "data": {
+          "pin": "PIN Code"
+        }
+      },
+      "pair_no_pin": {
+        "title": "Pairing",
+        "description": "Pairing is required for the `{protocol}` service. Please enter PIN {pin} on your Apple TV to continue."
+      },
+      "service_problem": {
+        "title": "Failed to add service",
+        "description": "A problem occurred while pairing protocol `{protocol}`. It will be ignored."
+      },
+      "confirm": {
+        "title": "Confirm adding Apple TV",
+        "description": "You are about to add the Apple TV named `{name}` to Home Assistant.\n\n**To complete the process, you may have to enter multiple PIN codes.**\n\nPlease note that you will *not* be able to power off your Apple TV with this integration. Only the media player in Home Assistant will turn off!"
+      }
+    },
+    "error": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "device_already_configured": "This device has already been configured. Please pick another device.",
+      "no_usable_service": "A device was found but could not identify any way to establish a connection to it. If you keep seeing this message, try specifying its IP address or restarting your Apple TV.",
+      "unknown": "Unexpected error! Check log file for further details.",
+      "auth": "Authentication error (you might have entered an invalid PIN code)."
+    },
+    "abort": {
+      "device_not_found": "Device could not be found on network, please try again.",
+      "already_configured": "Device is already configured!",
```

Comment:
There is some common strings in here, see #40578

Reply:
I will try to fix this.